oak/server/rust/oak_loader/src/main.rs

```diff
@@ -51,7 +51,8 @@ fn read_file(filename: &str) -> Result<Vec<u8>, Box<dyn std::error::Error>> {
     Ok(buffer)
 }
 
-fn main() -> Result<(), Box<dyn std::error::Error>> {
+#[tokio::main]
```

Comment:
Why do we need this right now?
The only this that needs `tokio` is a gRPC pseudo-node.
But it uses an instance of `tokio::runtime`:
https://github.com/project-oak/oak/blob/7e9f6cd424ab7729c4e12718ee1e8703e668eac5/oak/server/rust/oak_runtime/src/node/grpc_server.rs#L316-L319

Reply:
I don't know if runtime that is used by a gRPC node will work, if everything is already being run by a Tokio runtime.
But if it works, then don't see any problems with using `#[tokio::main]`